lib/express/crud/actions/index.js

```diff
@@ -11,6 +11,8 @@ module.exports = (model, defaultScope, params, arrayParams, sortBy, searchFields
     let perPage = req.query.perPage || 10
     perPage = Number(perPage)
 
+    const noPagination = req.query.noPagination || false
```

Comment:
Changes to this file should not be merged into master, they are from branch that was not merged.